torch/_functorch/partitioners.py

```diff
@@ -251,6 +249,14 @@ def _is_backward_state(node: fx.Node) -> bool:
     return node.op == "placeholder" and isinstance(node.meta.get("val"), BackwardState)
 
 
+def _has_tag_must_be_in_backward(node: fx.Node) -> bool:
+    return node.meta.get("partitioner_tag", None) == "must_be_in_backward"
+
+
+def _must_be_in_backward(node: fx.Node) -> bool:
+    return _has_tag_must_be_in_backward(node)
```

Comment:
`_has_tag_must_be_in_backward` seems unnecessary? (we can just inline it into `_must_be_in_backward`)

Reply:
My idea here is that in future _must_be_in_backward will accumulate multiple conditions, one of them will be `_has_tag_must_be_in_backward`